### ISSUE TITLE
Port `accuracy.cpp`

### ISSUE THREAD

**Issue by BogdanNitica99** [GabrielMajeri/anomrank-py#5]
accuracy.cpp tells how good it is at detecting anomalies over a dataset.

We can use [this function](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html). Maybe it helps.
